elastic/kibana
# [Infra UI] Consistency in the popover styling

Closes #165960 
## Summary

This PR sets all popups inside host view padding to size s (8px)

### Testing

- Open the host view and check the popup size (should be 8px - size s):
   <img width="1303" alt="image" src="https://github.com/elastic/kibana/assets/14139027/b3242de6-5644-4a1d-9f10-7b9a76269848">

- Open metadata and processes message popups - they should have the same padding size: 
   - Metadata Tab
   
     <img width="1002" alt="image" src="https://github.com/elastic/kibana/assets/14139027/e1c157aa-ced6-4a2e-9458-a38b99328276">

   - Overview Tab
   
      <img width="585" alt="image" src="https://github.com/elastic/kibana/assets/14139027/878c1325-28af-4b16-9ec4-93a0fc620c87">

   - Processes Tab
   
       <img width="1147" alt="image" src="https://github.com/elastic/kibana/assets/14139027/70dfc49f-ef1a-4786-bab4-7a3715af7f85">
